frontend/src/app/organization/organization-editor/organization-editor.component.css

```diff
@@ -1,17 +1,48 @@
-.mat-mdc-card {
-  margin: 1em;
-  max-width: 640px;
+mat-card {
+  margin: 2em;
 }
 
-.mat-mdc-form-field {
-  width: stretch;
+mat-card-header {
+  display: flex;
+  justify-content: center;
+  margin-bottom: 20px;
 }
 
-.mat-mdc-card-actions .mdc-button {
-  margin-left: 8px;
-  margin-bottom: 8px;
+mat-card-title {
+  flex: 1;
+  text-align: center;

```

Comment:
Could be left as left-align since the card itself also isn't centered on the page.